packages/glsp-playwright-core/src/integration/theia/po/theia-glsp-editor.po.ts

```diff
@@ -0,0 +1,23 @@
+import { normalizeId, OSUtil, TheiaEditor, urlEncodePath } from '@theia/playwright';
+import { join } from 'path';
+import type { TheiaGLSPApp } from './theia-glsp-app.po';
+
+export class TheiaGLSPEditor extends TheiaEditor {
+    constructor(filePath: string, app: TheiaGLSPApp) {
+        // shell-tab-code-editor-opener:file:///c%3A/Users/user/AppData/Local/Temp/cloud-ws-JBUhb6/sample.txt:1
```

Comment:
I like the examples, please add a comment above that they are examples, I was confused for a second ;-)

Reply:
Will be updated in the next PR with the Theia integration